source/in-use-encryption-tutorial.txt

```diff
@@ -0,0 +1,155 @@
+  Connection tab <in-use-encryption-tab>`.
+
+Create Your Encrypted Collection
+--------------------------------
+
+Once your deployment is connected using In-Use Encryption, create your collection 
+using Queryable Encryption. You can create a new database and collection or you 
+can create a new collection in an existing database.
+
+Procedure
+~~~~~~~~~
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click the :guilabel:`Create a Database` button or the 
+      :guilabel:`Create a Collection` button. 
+
+      Enter the name of the database and/or collection.
+
+   .. step:: Click the :guilabel:`Advanced Collection Options` drop down. 
+
+   .. step:: Check the :guilabel:`Queryable Encryption` box.
+
+   .. step:: Specify your :guilabel:`Encrypted Fields`. 
+
+      Change the ``path`` field value from ``encryptedField`` to the 
+      name of the field you want encrypted. 
+
+      .. figure:: /images/compass/example-encrypted-field.png
+         :figwidth: 450px
+         :alt: Encrypted Field Name 
+
+      Here, the encrypted field is the ``base`` field of the ``air_airlines`` 
+      data set.  
+
+      For more information, see :v6.0:`Encrypted Fields </core/queryable-encryption/fundamentals/encrypt-and-query/>`.
+
+   .. step:: (Optional) Specfiy :v6.0:`KMS Provider </core/queryable-encryption/fundamentals/kms-providers/>`.  
+
+   .. step:: (Optional) Specify :v6.0:`Key Encryption Key </core/queryable-encryption/fundamentals/keys-key-vaults/>`.
+
+   .. step:: Click :guilabel:`Create Database` or :guilabel:`Create Collection`.
+
+Import Your Data
+----------------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click on your collection on the left-hand navigation banner.
+
+      The collection has a :guilabel:`Queryable Encryption` badge next to 
+      its name to indicate that fields in that collection are encrypted.
+
+      .. figure:: /images/compass/qe-badge.png
+         :figwidth: 450 px
+         :alt: Queryable Encryption Badge 
+
+   .. step:: Click :guilabel:`Add Data`.
+
+   .. step:: Click :guilabel:`Import File`.
+
+   .. step:: Select File and Input File Type.
+
+   .. step:: Click :guilabel:`Import`.
+
+      Your imported collection is displayed in the document view. The specified 
+      encrypted field is marked by a key symbol next to the value.
+
+      .. figure:: /images/compass/encrypted-field.png
+         :figwidth: 450 px
+         :alt: Encrypted Field 
+
+      Here, the ``base`` field is marked with the key symbol.
+
+Enable and Disable In-Use Encryption
+------------------------------------
+
+You can enable and disable In-Use Encryption in your deployment. 
+
+When In-Use Encryption is **enabled**:
+
+- You can modify encrypted values.
+
+- You can insert documents and specified fields will be encrypted.
+
+When In-Use Encryption is **disabled**:
+
+- You can no longer modify encrypted values. |compass-short| displays the values 
```

Comment:
[suggestion]

Consider:

"You cannot modify encrypted values."

Reply:
Done.